django_thumbor/templatetags/thumbor_tags.py

```diff
@@ -27,3 +27,12 @@ def thumbor_url(image_url, **kwargs):
         kwargs['filters'] = _parse_filters(filters)
 
     return generate_url(image_url=image_url, **kwargs)
+
+@register.assignment_tag()
+def assign_thumbor_url(image_url, **kwargs):
```

Comment:
This is not related to the aliases feature right? Please send a separate PR next time ok?